nautobot/core/templates/generic/object_list.html

```diff
@@ -214,28 +214,11 @@ <h1>{% block title %}
                 </div>
                 {% endif %}
                 {% include table_template|default:'panel_table.html' %}
+
                 <div class="pull-left noprint">
+                    {% consolidate_bulk_action_buttons request content_type perms bulk_edit_url bulk_delete_url permissions %}
                     {% block bulk_buttons %}{% endblock %}
-                    {% if content_type.model_class.is_dynamic_group_associable_model and perms.extras.add_staticgroupassociation %}
-                        <button type="button"
-                                id="update_dynamic_groups_for_selected"
-                                data-toggle="modal"
-                                data-target="#dynamic_group_assignment_modal"
-                                data-objects="selected"
-                                class="btn btn-primary btn-sm">
-                            <span class="mdi mdi-group" aria-hidden="true"></span> Update Group Assignment
-                        </button>
-                    {% endif %}
-                    {% if bulk_edit_url and permissions.change %}
-                        <button type="submit" name="_edit" formaction="{% url bulk_edit_url %}{% if request.GET %}?{{ request.GET.urlencode }}{% endif %}" class="btn btn-warning btn-sm">
-                            <span class="mdi mdi-pencil" aria-hidden="true"></span> Edit Selected
```

Comment:
Should this PR also address the corresponding edit-all/delete-all buttons in this same file or is that tracked by a separate issue?

Reply:
@lampwins do we want to change these buttons as part of this issue?

![image](https://github.com/nautobot/nautobot/assets/75227981/fbabd616-a898-49f9-b349-883b280771f6)

Reply:
No, out of scope.